packages/core/src/application.ts

```diff
@@ -207,3 +211,14 @@ export interface ApplicationConfig {
 
 // tslint:disable-next-line:no-any
 export type ControllerClass = Constructor<any>;
+
+/**
+ * Type description for `package.json`
+ */
+export interface PackageJson {
+  name: string;
+  version: string;
+  description: string;
+  // tslint:disable-next-line:no-any
+  [name: string]: any;
```

Comment:
If this data object is typically loaded from JSON, then I think we should restrict the allowed property types to `number | string | object | array | null`. JSON does not support functions and Dates for example. The current type definition makes it easy to accidentally treat a package.json property as a function and call it.

Reply:
> If this data object is typically loaded from JSON, then I think we should restrict the allowed property types to `number | string | object | array | null`. JSON does not support functions and Dates for example. The current type definition makes it easy to accidentally treat a package.json property as a function and call it.

This comment hasn't been addressed nor responded to, PTAL.

Reply:
Fixed